core/src/main/java/org/elasticsearch/search/aggregations/Aggregations.java

```diff
@@ -98,4 +107,35 @@ public final boolean equals(Object obj) {
     public final int hashCode() {
         return Objects.hash(getClass(), aggregations);
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        if (aggregations.isEmpty()) {
```

Comment:
`aggregations` is not initialized with emptyList() anymore, I think this might cause an issue here